packages/components/src/List/ListComposition/Manager.component.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ListContext } from './context';
+
+export default class Manager extends React.Component {
+	static displayName = 'List.Manager';
+	static propTypes = {
+		children: PropTypes.node,
+
+		displayMode: PropTypes.string, // eslint-disable-line react/no-unused-prop-types
+		onDisplayModeChange: PropTypes.func,
+	};
+
+	static getDerivedStateFromProps(props) {
+		return { collection: props.collection };
```

Comment:
It is bad to just copy props into state, but this is to prepare the structure for next iterations where we will have built-in sorting and paging for uncontrolled features.